src/Components/Main/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import data from "../../data";
+class Main extends Component {
+  state = {
+    score: 0,
+    timer: 60,
+    counter: 0,
+    currentImg: data[0].img,
+    currentName: data[0].name,
+    answer: ""
+  };
+
+  compare = event => {
+    const { value } = event.target;
+    const { currentName, counter, score } = this.state;
+    this.setState({ answer: value }, () => {
```

Comment:
The syntax here is wrong, this should take prevState (can be destructured) but shouldn't have an extra arrow function.